### ISSUE TITLE
Empty tag added when I add the first element

### ISSUE THREAD

**Issue by fcovasquez** [kartik-v/yii2-widget-select2#232]
Hello

I'm having the following issue when I use Select2 widget with tags support.

When I add the first element an extra empty tag is added automatically as showed in the following image
![tagproblem](https://user-images.githubusercontent.com/2367363/32249405-b02a40b2-be67-11e7-89e4-b615c98de9a1.png)

The code I'm using is the following:

`
echo $form->field($model, 'email')->widget(\kartik\widgets\Select2::classname(), [
    'data' => $emails,
    'options' => [
        'placeholder' => 'Seleccione un correo ...',
        'multiple' => true,
    ],
    'pluginOptions' => [
        'tags' => true,
        'tokenSeparators' => [',', ' '],
    ],
]);`

If I add an existing element (existing in $emails array) the empty tag is added at the end of the list. If I add a new tag the empty tag is added at the beggining of the list

If I delete manually the extra tag I can continue without problem, but is annoying to the users. I believe that the problem appears with the last update of the widget (v 2.1.0)

I have reproduced the problem in Google Chrome and in Microsoft Edge.

Thanks in advanced

**Comment by kartik-v** [comment]
You may need to check your code. Can you cross check with the demo?

Cannot reproduce the same on the [widget demo page for tags](http://demos.krajee.com/widget-details/select2#usage-tags).

**Comment by fcovasquez** [comment]
Hi,

I checked the code. It was my problem. The code that generates the array of possible emails also was adding an empty element in the array. That was happening because I was using the explode method to convert a comma separated string into an array and when the string was empty the explode method generates only an empty element in the array array([0] =>). When I changed that behavior the problem was solved.

I comment this just in case It could help someone else.

Thanks
